python/lsst/obs/cfht/ingest.py

```diff
@@ -46,47 +45,13 @@
            }
 
 
+@deprecated(reason="MegaPrime no longer requires a specialist gen3 ingest task. Please use the default."
+            " Will be removed after v22.",
+            category=FutureWarning,
+            version="21.0")
 class MegaPrimeRawIngestTask(lsst.obs.base.RawIngestTask):
     """Task for ingesting raw MegaPrime multi-extension FITS data into Gen3.
     """
-    def extractMetadata(self, filename: ButlerURI) -> RawFileData:
```

Comment:
Should this have an explicit `pass` for readability?

Reply:
Python doesn't require it. The `pass` seems superfluous to me.